code/modules/mob/living/silicon/silicon.dm

```diff
@@ -85,13 +85,13 @@
 	modularInterface.saved_identification = real_name || name
 	if(istype(src, /mob/living/silicon/robot))
```

Comment:
these should probably be in their own respective files tbh, not all in `silicon.dm`